app/config/config_dev.yml

```diff
@@ -14,15 +14,18 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
-            path:  %kernel.logs_dir%/%kernel.environment%.log
-            level: debug
+            type:   stream
+            path:   %kernel.logs_dir%/%kernel.environment%.log
+            level:  debug
+        console:
+            type:   console
+            bubble: false
```

Comment:
Hello. 

I just hit an issue here with bubble and I would like to discuss about it

I was trying to debug my application, and so I added a new handler to push all my logs to the Elasticsearch/Heka/Kibana stack (to get a better visualisation / filter system for my logs...) . I did not really think about the order of the handlers and so I added my new handler after the previous handlers. And so I ended-up with the following config:

```
monolog:
    handlers:
        main:
            type:   stream
            path:   "%kernel.logs_dir%/%kernel.environment%.log"
            level:  debug
            channels: ["!event", "!consumer_status", "!beberlei_metrics"]
        console:
            type:   console
            bubble: true
            channels: ["!doctrine", "!event", "!consumer_status", "!beberlei_metrics"]
        heka:
            type: socket
            level:  debug
            connection_string: "udp://heka.lxc:5565"
            timeout: 1
            connection_timeout: 1
            formatter: monolog.formatter.json
```

After that, I tried to debug my code and noticed I missed lot of logs. I try to understand why (is it an issue with ES or heka or kibana ? ; It is an issue with my code ? ; etc ?). And I finally understood my issue.

So finally, I really think the console handler should bubble all logs, because It's "just" a nice to have feature (A very good one BTW, thanks again). But it should not "eat" logs. All handlers registered after the handler should also be able to get all logs. So, IMHO, we should turn on the bubble here.

WDTY ?

ping @Tobion @Seldaek